scripts/update_ontology_terms/utils.py

```diff
@@ -0,0 +1,93 @@
+import requests
+import json
+import os
+from submission import Submission
+import subprocess
+
+INGEST_API_TOKEN = os.environ['INGEST_API_TOKEN']
+INGEST_API = os.environ.get('INGEST_API_URL', 'https://api.ingest.archive.data.humancellatlas.org')
+INGEST_API.strip('/')
+DEFAULT_HEADERS = {'Content-type': 'application/json'}
+
+def read_lines(file: str):
+    with open(file, 'r') as f:
+        for l in f:
+            yield l
+    
+def get_protocol(protocol_uuid: str):
```

Comment:
Should such functions for querying ingest api be in some shared module? It seems we are writing those repeatedly. Isn't this what the ingest client is for?